src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -183,6 +183,11 @@ export default (accessToken, userID, opts) => {
       },
       { method: "POST" }
     ),
+    myCollectionCreateImageLoader: gravityLoader(
+      (id) => `artwork/${id}/image`,
+      {},
```

Comment:
(nonblocking) Just noting that all of the other MC endpoints require this auth object here:
```
 user_id: userID,
private: true,
```

Reply:
Unless I'm missing something, I believe these additional params are not required for this particular endpoint.